gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/public/lib/foundation/js/foundation/foundation.dropdown.js

```diff
@@ -0,0 +1,326 @@
+;(function ($, window, document, undefined) {
```

Comment:
I don't believe this file is used in our application - we should remove it from the commit